Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -319,16 +319,14 @@ static void HandleQuery()
 
 static void HandleSetThread()
 {
-  if (memcmp(s_cmd_bfr, "Hg0", 3) == 0 || memcmp(s_cmd_bfr, "Hc-1", 4) == 0 ||
-      memcmp(s_cmd_bfr, "Hc0", 3) == 0 || memcmp(s_cmd_bfr, "Hc1", 3) == 0)
+  if (memcmp(s_cmd_bfr, "Hc-1", 4) == 0 || memcmp(s_cmd_bfr, "Hc1", 3) == 0)
```

Comment:
Can you explain why Hc0 was removed?

> Several packets and replies include a thread-id field to identify a thread. Normally these are positive numbers with a target-specific interpretation, formatted as big-endian hex strings. A thread-id can also be a literal ‘-1’ to indicate all threads, or ‘0’ to pick any thread.

Is "any thread" not a valid option?

Reply:
huh, I actually didn't read that part, I assumed 0 was wrong because gdb wasn't actually inquiring about it, but if the specs says it's fine, sure, I'll put it back (also I actually thought this was a dash and not a -1...oops)